From 1da3e58d20d8e5b2d3aa33c613830c5aa33683ef Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Tue, 17 Dec 2019 09:59:38 +0200
Subject: [PATCH] [SW-6739] arcboot: reduce make verbosity

disable output of synopsys make, only if there are issues it will
print them out

Change-Id: Id7b70e9c4546c4e266a8479e471ef6e3a2c718c2
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/Makefile b/Makefile
index c2dbb83..a018d5b 100644
--- a/Makefile
+++ b/Makefile
@@ -45,14 +45,14 @@
 	@$(MAKE) arcboot.bin arcboot_rom.bin
 
 libsb libellipsys:
-	@$(MAKE) -C sdk $@ SB_SIGN_MODE=rsa_std
+	@$(MAKE) -C sdk $@ SB_SIGN_MODE=rsa_std > /dev/null
 sbtool:
-	@$(MAKE) -C sdk $@
+	@$(MAKE) -C sdk $@ > /dev/null
 %.bin: %
 	@$(OBJCOPY) -O binary -R .note.gnu.build-id -R .fw_sram -R .test* $< $@
 
 arcboot: $(ARCBOOT_OBJECTS)
-	$(CC) $^ $(LDFLAGS) -o $@
+	@$(CC) $^ $(LDFLAGS) -o $@
 arcboot_rom: $(ARCBOOT_ROM_OBJECTS)
 	@$(CC) $^ $(LDFLAGS) -o $@
 
